README.md

```diff
@@ -71,6 +72,42 @@ new_task = create_task(
 update_task(new_task.duid, status_title="Done")
 ```
 
+## Using the Python Library in AWS Lambda Functions
```

Comment:
We already have our well-built `wheel` that is going to work on many platforms. But this `wheel` comes with some Python dependencies like `regex` and when **`user builds a deployment package for the wrong Python version or arch that differs from one setup in AWS Lambda function`**, this will just fail with an exception.

> What do common packages like numpy do for example?

This package can be installed either from `wheel` (if such `wheel` exists for the given version of Python and arch) or just built for the given version of Python and arch (usually using Docker). A mismatched version of Python or arch with ones used in runtime env will lead to the same (failure) result for `numpy` or any other library.
<img width="1428" alt="Screenshot 2024-12-25 at 19 39 37" src="https://github.com/user-attachments/assets/7a49f54d-c956-4026-9ae6-f64187fc3bc2" />

Reply:
> Do people have to follow a complicated process like this if they want to use numpy in a lambda?

Yes, either following these steps I've provided or using layers in AWS Lambda (built by users or someone else).

Reply:
OK, I understand most of this. But I have some questions still, sorry
1. Does `numpy` (continuing to use that as an example) provide a prebuilt wheel for some/all of the standard pythons + architecture?
2. How much effort would it take to generate these `whl`s?

Reply:
I would still like to try to make this easier for users than it is. It seems like the solution in this PR is still pretty complex. Why can't we generate the wheels?

Reply:
> OK, I understand most of this. But I have some questions still, sorry
> 
> 1. Does `numpy` (continuing to use that as an example) provide a prebuilt wheel for some/all of the standard pythons + architecture?

Yes, it does, please see [this](https://pypi.org/project/numpy/2.2.1/#files) `numpy` release.

> 2. How much effort would it take to generate these `whl`s?

If we really need this, extending the build script may take a day or so. But if this user still uses the wrong target runtime's arch or Python version, the result will be completely the same (failure).

> Why can't we generate the wheels?

We can, but it will not change anything when the user creates AWS Lambda deployment package using the wrong wheels. However, if we have these wheels, the instructions could be simplified (if so) to something like below:
- For AWS Lambda `x86_64` arch runtime with `Python 3.12`, download `link-to-some-whl` and put to your deployment package folder
...
@zswaff do you expect something like that about these instructions?


https://docs.aws.amazon.com/lambda/latest/dg/python-package.html
When this user creates the deployment package without using Docker, most probably, arch and / or Python version of the host machine won't match ones of AWS lambda runtime. This is the actual root cause.